packages/instrumented-component-lifecycle/README.md

```diff
@@ -0,0 +1,23 @@
+# `@bugsnag/instrumented-component-lifecycle`
+
+A HOC component for instrumenting BugSnag Performance spans.
+
+## Usage
+
+```typescript
+import BugsnagPerformance from "@bugsnag/browser-performance"
+import withInstrumentedComponentLifecycle from "@bugsnag/instrumented-component-lifecycle"
```

Comment:
@tomlongridge do you have any thoughts about the name of the component and/or the package here? I wonder if it should have the word 'react' in there as well?